docs/migration/migrate_7_3.asciidoc

```diff
@@ -9,11 +9,21 @@ your application to Kibana 7.3.
 
 See also <<release-highlights-7.3.0, release highlights>> and <<release-notes-7.3.0, release notes>>.
 
-coming[7.3.0]
-
 //NOTE: The notable-breaking-changes tagged regions are re-used in the
 //Installation and Upgrade Guide
 
 //tag::notable-breaking-changes[]
 
-// end::notable-breaking-changes[]
\ No newline at end of file
+[float]
+[[breaking_73_data_frames_date_format]]
+=== Removed support for date histogram format specifications in {dataframe-transforms}
+
+*Details:* If your {dataframe-transform} groups your data by using a date
+histogram, the `format` property is no longer supported.
+
+*Impact:* {dataframe-transforms-cap} that were created in 7.2 and use date
+histograms might no longer work in 7.3. If you run into issues starting a
+transform, recreate it by copying the `pivot` part of the configuration into the
```

Comment:
Won't the pivot part of the configuration contain the date histogram (and therefore the problematic format property)?
Would it be appropriate to say something like "clone the transform by copying the pivot configuration details--except the format property--into the advanced editor of the wizard"

Reply:
The advanced editor in the UI will automatically remove the unsupported `format` attribute once the use clicks `Apply`, I added `The advanced editor will
remove the unsupported attribute once the configuration gets applied.` to clarify.